### PR TITLE
Update to Spring Cloud Stream snapshots

### PR DESCRIPTION
**Pull request by dsyer**
There are some bug fixes it would help to pick up (log hygiene if
nothing else).

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
-		<spring-cloud.version>Edgware.RELEASE</spring-cloud.version>
+		<spring-cloud.version>Edgware.BUILD-SNAPSHOT</spring-cloud.version>
```
Comment (trisberg) [inline pom.xml:22]: Should we bump this to "Edgware.SR1" before our 0.0.3 release?

File: src/test/java/io/projectriff/functions/WordCounter.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/*
```
Comment (trisberg) [inline src/test/java/io/projectriff/functions/WordCounter.java:1]: What is this file for? Do you use it for your testing?

Comment (dsyer) [reply src/test/java/io/projectriff/functions/WordCounter.java:1]: Yes. It makes a better test of “real” data. Should probably fold it into the integration tests at some point.

File: pom.xml (hunk 1 above)
Comment (dsyer) [reply pom.xml:22]: Yeah, that would make sense.

Comment (trisberg) [reply pom.xml:22]: I can change that on merge

**Comment by trisberg** [comment]
Rebased and merged as commit 22eb064a7e67554e8328b64633e197757e533c01

**Pull request closed without merging**